mllib/src/main/scala/org/apache/spark/ml/regression/GeneralizedLinearRegression.scala

```diff
@@ -782,8 +782,12 @@ object GeneralizedLinearRegression extends DefaultParamsReadable[GeneralizedLine
 
     override def variance(mu: Double): Double = mu
 
+    private def ylogy(y: Double, mu: Double): Double = {
+      if (y == 0) 0.0 else y * math.log(y / mu)
+    }
+
```

Comment:
Another `ylogy` implementation in `Binomial`. Can you move this code to `object GeneralizedLinearRegression` and make it private to this package?

Reply:
Thanks so much for the quick review. I have moved the `ylog` implementation to `object GeneralizedLinearRegression`. One quick question here: I am not sure I have fully understood why this is the right place for `ylog`? Thanks!

Reply:
Any suggestion to avoid the duplicated code? Let's followup this later if you have an idea.